### PR TITLE
Refactoring and eslint rules

### PR DESCRIPTION
**Pull request by MrRamka**
- Fixed eslint rules
- Removed old UI manager
- Removed unused components

### CODE REVIEW

File: src/app/components/colors/color_dimension_input.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  updateState: (state: number) => void;
+}
+
+export class ColorDimensionInput extends React.Component<
+  ColorDimensionInputProps,
+  Record<string, unknown>
+> {
+  render() {
+    return (
+      <>
```
Comment (jrthompson33) [inline src/app/components/colors/color_dimension_input.tsx:29]: Why is the element missing a tag?

File: src/app/components/colors/color_hex_input.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  render() {
+    const currentColor = this.props.state.desc.toRGB(
+      this.props.state.x1,
+      this.props.state.x2,
+      this.props.state.x3
+    );
+    const rgb = { r: currentColor[0], g: currentColor[1], b: currentColor[2] };
+
+    return (
+      <>
```
Comment (jrthompson33) [inline src/app/components/colors/color_hex_input.tsx:26]: Same here, missing a tag.

Comment (MrRamka) [reply src/app/components/colors/color_hex_input.tsx:26]: This is short syntax of react fragment. 
https://reactjs.org/docs/fragments.html

File: src/app/components/colors/color_dimension_input.tsx (hunk 1 above)
Comment (MrRamka) [reply src/app/components/colors/color_dimension_input.tsx:29]: This is short syntax of react fragment.
https://reactjs.org/docs/fragments.html

File: src/app/views/panels/widgets/controls/image.tsx [hunk 3]
```diff
@@ -90,19 +89,17 @@ export class InputImage extends ContextedComponent<
...
         .map((x) => x.trim())
         .filter((x) => !x.startsWith("#"));
-      ImageUploader.ParseURIs(uris)
-        .then((r) => {
-          this.emitOnChange(r);
-        })
-        .catch(() => {});
+      ImageUploader.ParseURIs(uris).then((r) => {
+        this.emitOnChange(r);
+      });
```
Comment (jrthompson33) [inline src/app/views/panels/widgets/controls/image.tsx:94]: Consider adding error catch here.

**Review by jrthompson33 — COMMENTED** [review]
PR looks good! Please take a look at the comments for minor changes.

File: src/app/views/panels/widgets/controls/image.tsx (hunk 3 above)
Comment (MrRamka) [reply src/app/views/panels/widgets/controls/image.tsx:94]: Fixed

**Review by bongshin — APPROVED** [review]

**Pull request merged**